applications/launchpad_v2/src/utils/ExpertViewUtils.ts

```diff
@@ -0,0 +1,41 @@
+import { useSpring } from 'react-spring'
+import { ExpertViewType } from '../store/app/types'
+
+/**
+ * Helpers for the ExpertView
+ */
+class ExpertViewUtils {
```

Comment:
what's the point of a class if you only have static members?
these should be just `export const ... = ` on the module level or maybe
export default { convertEx, drawerAnim... }

Reply:
It groups functions nicely. So then in the code we can use:

```
ExpertViewUtils.convertXtoY(...)
```

instead of:

```
convertXtoY(...)
```

But I'm open to use plain functions.

Reply:
as I wrote on slack, the basic mechanism of code reuse and code grouping in node is a module
most of the time you shouldn't need anything else

if you want to call it explicitly with `ExpertViewUtils.someUtil`, you can `import * as ExpertViewUtils` and you can call these methods explicitly

this just seems like unnecessary layers of indirection, which can be dealt with using primitive mechanisms of node